app/src/main/res/values/styles.xml

```diff
@@ -125,7 +125,7 @@
   </style>
 
   <style name="backgroundCardStyleLight">
-    <item name="android:background">@drawable/white_card</item>
+    <item name="android:background">?attr/backgroundCardColor</item>
```

Comment:
Missing the attributes defined in backgroundCardStyleDark for the light theme. Right now the selection feedback doesn't work for white theme.

Reply:
Also, the editorial card and editors' choice cards have the wrong background color